guides/rails_guides/markdown/renderer.rb

```diff
@@ -58,18 +66,20 @@ def convert_footnotes(text)
           end
         end
 
-        def class_for(code_type)
+        def lexer_language(code_type)
           case code_type
-          when "ruby", "sql", "plain", "js", "yaml"
+          when "js", "http", "html", "ruby", "sql", "yaml", "yml"
             code_type
           when "erb", "html+erb"
             "erb"
           when "html"
-            "xml" # HTML is understood, but there are .xml rules in the CSS
+            "html"
```

Comment:
This is already covered by the first branch.

Reply:
I've removed this branch